### PR TITLE
Add some schema helper methods for SessionPool

### PR DESCRIPTION
**Pull request by haoxins**
## What type of PR is this?
- [ ] bug
- [x] feature
- [ ] enhancement

#### Issue(s) number: 

#311

### CODE REVIEW

File: label.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	Type    string `nebula:"Type"`
+	Null    string `nebula:"Null"`
+	Default string `nebula:"Default"`
+	Comment string `nebula:"Comment"`
+}
+
+type LabelFieldSchema struct {
+	Field string
+	Type  string
+	Null  bool
```
Comment (Nicole00) [inline label.go:18]: IsNull is better for bool var name?

File: label.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+)
+
+type Label struct {
+	Field   string `nebula:"Field"`
+	Type    string `nebula:"Type"`
+	Null    string `nebula:"Null"`
+	Default string `nebula:"Default"`
+	Comment string `nebula:"Comment"`
+}
+
```
Comment (Nicole00) [inline label.go:14]: we can define `LabelName` here to represent the result of `show tags`/`show edges`.

File: session_pool.go [hunk 3]
```diff
@@ -272,6 +272,103 @@ func (pool *SessionPool) GetTotalSessionCount() int {
...
+	errCode := rs.GetErrorCode()
+	if errCode != ErrorCode_SUCCEEDED {
+		errMsg := rs.GetErrorMsg()
+		return nil, fmt.Errorf("Fail to execute query. %s", errMsg)
+	}
+
+	return rs, nil
+}
+
+type _Label struct {
```
Comment (Nicole00) [inline session_pool.go:290]: move it to label.go

File: label.go [hunk 4]
```diff
@@ -0,0 +1,78 @@
+package nebula_go
```
Comment (Nicole00) [inline label.go:1]: please add copyright for new files.

File: session_pool.go (hunk 3 above)
Comment (haoxins) [reply session_pool.go]: done

File: label.go (hunk 4 above)
Comment (haoxins) [reply label.go:8]: done

File: label.go (hunk 1 above)
Comment (haoxins) [reply label.go]: I change it to `Nullable`

File: label.go (hunk 2 above)
Comment (haoxins) [reply label.go:21]: done

**Review by Nicole00 — APPROVED** [review]
Great work！ Thanks for your contribution again！

**Pull request merged**